### PR TITLE
Added test for the skip function

### PR DESCRIPTION
**Pull request by ShriprajwalK**

### CODE REVIEW

File: test/secretbuffer.jl [hunk 1]
```diff
@@ -80,4 +80,15 @@ using Test
         @test read(sb, String) == "\xff\xff\xff"
         shred!(sb)
     end
+    @testset "testing the skip function" begin
+    sb = SecretBuffer("computer")
```
Comment (StefanKarpinski) [inline test/secretbuffer.jl]: Indentation.

File: test/secretbuffer.jl [hunk 2]
```diff
@@ -80,4 +80,15 @@ using Test
...
+    skip(sb, 2)
+    a = position(sb)
+    seek(sb,0)
```
Comment (StefanKarpinski) [inline test/secretbuffer.jl]: Spacing. I would suggest spaces after commas in function calls here.

File: test/secretbuffer.jl (hunk 1 above)
Comment (ShriprajwalK) [reply test/secretbuffer.jl]: Sorry about that! Changed it in the new commit!

File: test/secretbuffer.jl (hunk 2 above)
Comment (ShriprajwalK) [reply test/secretbuffer.jl]: Done! Changed it in the new commit!

**Comment by ShriprajwalK** [comment]
I've done a few changes to make the code better. Good to go from my side!

**Review by vtjnash — APPROVED** [review]

**Pull request merged**